AnalysisNtuple/makeAnalysisNtuple.C

```diff
@@ -386,14 +391,99 @@ makeAnalysisNtuple::makeAnalysisNtuple(int ac, char** av)
+    	if (year=="2018") {uncFile = TFile::Open("SmearingScalingFiles/electron_2018_ScaleDo.root","READ");} 
+		TH2F *uncHist  = (TH2F*) uncFile->Get("pt_eta");
+    	selector->uncHist = uncHist;
+
+    }
+    if(systematicType=="elescale_up")   {
+    	elescale012_g=2;selector->elescaleLevel=2; isSystematicRun = true;
+        TFile *uncFile = new TFile();
+   		if (year=="2016") {uncFile = TFile::Open("SmearingScalingFiles/electron_2016_ScaleUp.root","READ");} 
+    	if (year=="2017") {uncFile = TFile::Open("SmearingScalingFiles/electron_2017_ScaleUp.root","READ");} 
+    	if (year=="2018") {uncFile = TFile::Open("SmearingScalingFiles/electron_2018_ScaleUp.root","READ");} 
+		TH2F *uncHist  = (TH2F*) uncFile->Get("pt_eta");
+    	selector->uncHist = uncHist;
+
+    }
+
+
+
+    if(systematicType=="phosmear_down") {
+    	phosmear012_g=0;selector->phosmearLevel=0; isSystematicRun = true;
+        TFile *uncFile = new TFile();
+   		if (year=="2016") {uncFile = TFile::Open("SmearingScalingFiles/photon_2016_ResDo.root","READ");} 
+    	if (year=="2017") {uncFile = TFile::Open("SmearingScalingFiles/photon_2017_ResDo.root","READ");} 
+    	if (year=="2018") {uncFile = TFile::Open("SmearingScalingFiles/photon_2018_ResDo.root","READ");} 
+		TH2F *uncHist  = (TH2F*) uncFile->Get("pt_eta");
+    	selector->uncHist = uncHist;
+
+    }
+    if(systematicType=="phosmear_up")   {
+    	phosmear012_g=2;selector->phosmearLevel=2; isSystematicRun = true;
+        TFile *uncFile = new TFile();
+    	if (year=="2016") {uncFile = TFile::Open("SmearingScalingFiles/photon_2016_ResUp.root","READ");} 
+    	if (year=="2017") {uncFile = TFile::Open("SmearingScalingFiles/photon_2017_ResUp.root","READ");} 
+    	if (year=="2018") {uncFile = TFile::Open("SmearingScalingFiles/photon_2018_ResUp.root","READ");} 
+		TH2F *uncHist  = (TH2F*) uncFile->Get("pt_eta");
+    	selector->uncHist = uncHist;
+
+    }
+
+
+    if(systematicType=="phoscale_down") {
+    	phoscale012_g=0;selector->phoscaleLevel=0; isSystematicRun = true;
+        TFile *uncFile = new TFile();
+    	if (year=="2016") {uncFile = TFile::Open("SmearingScalingFiles/photon_2016_ScaleDo.root","READ");} 
+    	if (year=="2017") {uncFile = TFile::Open("SmearingScalingFiles/photon_2017_ScaleDo.root","READ");} 
+    	if (year=="2018") {uncFile = TFile::Open("SmearingScalingFiles/photon_2018_ScaleDo.root","READ");} 
+		TH2F *uncHist  = (TH2F*) uncFile->Get("pt_eta");
+    	selector->uncHist = uncHist;
+
+
+    } 
+    if(systematicType=="phoscale_up")   {
+    	phoscale012_g=2;selector->phoscaleLevel=2; isSystematicRun = true;
+        TFile *uncFile = new TFile();
+    	if (year=="2016") {uncFile = TFile::Open("SmearingScalingFiles/photon_2016_ScaleUp.root","READ");} 
+    	if (year=="2017") {uncFile = TFile::Open("SmearingScalingFiles/photon_2017_ScaleUp.root","READ");} 
+    	if (year=="2018") {uncFile = TFile::Open("SmearingScalingFiles/photon_2018_ScaleUp.root","READ");} 
```

Comment:
Put these root files in the MuEGammaScaleFactors directories, so they are stored in the same place as the rest of the correction information, and add these to git as well.